core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -375,7 +408,40 @@ public static Duration getDuration(PropertyKey key) {
    *         including all default properties
    */
   public static Map<String, String> toMap() {
-    Map<String, String> map = toRawMap();
+    return toMapInternal(false);
+  }
+
+  /**
+   * @return a view of the display values for resolved properties represented by this configuration,
+   *         including all default properties
+   */
+  public static Map<String, String> toDisplayMap() {
```

Comment:
I feel we have too many `toMap` like methods now. How about consolidating them into one `toMap` but use different config value or args to control (1) raw or not, (2) display or not? I can see there are four different combinations and it doesn't make sense to have four `toMap` methods. And I believe code will be also simplified too.

Reply:
I understand it can be overwhelming to have many methods doing similar work. The problem with having one single method is that it could be confusing to read and understand difference between `toMap(true, false)` and `toMap(false, true)` without looking at the definition of the method. Unfortunately Java does not have a named parameter so we could not do `toMap(displayValue=true, rawValue=false)`.

Reply:
that's what I am proposing by using config. one can have `toMap(ToMapConfig.defaults().raw(true))` or `toMap(ToMapConfig.defaults().asDisplayValue(true))`

Reply:
Per our discussion, unified methods into `toMap(ValueOptions)`